kmod/src/dir.c

```diff
@@ -1694,6 +1699,16 @@ static int scoutfs_rename_common(struct inode *old_dir,
 		goto out_unlock;
 	}
 
+	if ((old_inode != NULL) && scoutfs_inode_worm_denied(old_inode)) {
```

Comment:
Might as well combine these two, they're so similar that it's not confusing to see them mashed together with some more () || ().